hack/release.sh

```diff
@@ -267,7 +267,7 @@ EOF
 
 	# Add the DEB package to the APT repo
 	DEBFILE=bundles/$VERSION/ubuntu/lxc-docker*.deb
```

Comment:
We'll just get another `shellcheck` drive-by that tries to add them back, and we'll forget this whole conversation. :wink: